spring-boot/admin/src/main/java/eu/coatrack/admin/service/GitService.java

```diff
@@ -64,8 +57,23 @@
     @Value("${ygg.admin.api-base-url-for-gateway}")
     private String adminApiBaseUrlForGateway;
 
+    private String ConvertSelectedSensitiveHeadersToZuulConfigFormat(Proxy proxy) {
+
+        String sensitiveHeadersConcatenated = "";
+        if (!proxy.isForwardCookie()) {
+            sensitiveHeadersConcatenated = sensitiveHeadersConcatenated.concat("Cookie,");
+        }
+        if (!proxy.isForwardSetCookie()) {
+            sensitiveHeadersConcatenated = sensitiveHeadersConcatenated.concat("Set-Cookie,");
+        }
+        if (!proxy.isForwardAuthorization()) {
+            sensitiveHeadersConcatenated = sensitiveHeadersConcatenated.concat("Authorization,");
+        }
+
+        return "zuul.sensitiveHeaders: " + sensitiveHeadersConcatenated;
```

Comment:
We need to discuss about this, so in order for this to work, we need to use the Zullfilter dependency, so we need to make sure that after the PR122 the Zullfilter dependency is not removed